src/Backend/Modules/Pages/Actions/Edit.php

```diff
@@ -253,6 +253,35 @@ private function loadForm()
         $this->frm->addImage('image');
         $this->frm->addCheckbox('remove_image');
 
+        // page auth related fields
+        // check if profiles module is installed
+        if (BackendModel::isModuleInstalled('Profiles')) {
+            // check data for auth_required
+            $auth_required = false;
+            if (isset($this->record['data']['auth_required']) && $this->record['data']['auth_required']) {
+                $auth_required = true;
+            }
+            // add checkbox for auth_required
+            $this->frm->addCheckbox('auth_required', $auth_required);
```

Comment:
How would you do this? Here we have to check for two things, if it exists and if it is true?